index.js

```diff
@@ -130,6 +130,12 @@ Transformer.prototype.can = function (method) {
 
 Transformer.prototype.compile = function (str, options) {
   if (!this.can('compile')) {
+    if (this.can('render')) {
+      var _this = this;
+      return tr.normalizeFn(function (locals) {
+        return tr.normalize(_this._tr.render(str, options, locals));
```

Comment:
@TimothyGu Does this look correct? Normalize the `function(locals)`, as well as the `render()` return?